app/src/main/java/ar/com/wolox/android/login/ui/training/LoginPresenter.java

```diff
@@ -0,0 +1,72 @@
+package ar.com.wolox.android.login.ui.training;
+
+import ar.com.wolox.android.login.utils.UserSession;
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+import javax.inject.Inject;
+
+public class LoginPresenter extends BasePresenter<ILoginView> {
+
+    // Constants
+    public static final String TAG = "LoginPresenter";
+    public static final int NUMBER_MAX = 1000;
```

Comment:
Are you using those numbers? It looks like it is old code.

Reply:
Yes, it's old code, but i didn't know. It's solved in the next branch. (https://github.com/wolox-training/dm-android/blob/login-frontend-2/app/src/main/java/ar/com/wolox/android/training/ui/login/LoginPresenter.java#L10)